src/room007/models/model.py

```diff
@@ -38,6 +44,43 @@ def option_combination_name(options):
         else:
             return [('||| no parameter combinations |||', {option_name: option.default for option_name, option in self.options.items()})]
 
+
+class FeatureManager(object):
+    save = False
+    def __init__(self, fun=None):
+        self.fun = fun
+
+    def __call__(self, *args, **kwargs):
+        data = args[0]
+        feature_name = self._get_feature_name(args, kwargs)
+        if feature_name in data:
+            features = data[feature_name].apply(lambda x: eval(x) if '[' in x else x)
```

Comment:
Is it about the eval?
I would like to remove this completely in fact by having a better way to store list in the CSV, but I think we should agree together on this how we want to do this. If we agree on something we can completely remove this ugly code.